src/locale/locales/fr/messages.po

```diff
@@ -1004,6 +1012,10 @@ msgstr "Sous-titres (.vtt)"
 msgid "Captions & alt text"
 msgstr "Sous-titres et texte alt"
 
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:369
+msgid "Celebrating {0} users"
+msgstr "Célébration de {0} de comptes"
```

Comment:
Translating a `-ing` form as a noun is good most of the time, but in _that_ case, it would make it lose its force drastically. Let's be more "active".

```suggestion
msgstr "Célébrons les {0} comptes"
```